.vscode/settings.json

```diff
@@ -1,11 +1,4 @@
 {
-  "ruby.useBundler": true,
-  "ruby.useLanguageServer": true,
-  "ruby.lint": {
-    "rubocop": {
-      "useBundler": true
-    }
-  },
-  "ruby.format": "rubocop",
-  "ruby.intellisense": "rubyLocate"
+  "rubyLsp.rubyVersionManager": "mise",
```

Comment:
Is this setting necessary? because we also have a devcontainer environment which doesn't rely on `mise` and it will instead throw an error.

Reply:
so this is necessary -- without telling rubyLsp that `mise` is ruby's version manager, rubyLsp tries to use `asdf` when running any of the rubyLsp tasks in `launch.json`:
![Screen Shot 2024-04-02 at 9 30 33 AM](https://github.com/forem/forem/assets/13890477/0b7192ef-bd71-470b-9a73-1d418457b823)

Reply:
i don't wanna introduce an error in `devcontainer` tho 🤔 
i wonder if it's possible to override the rubyLsp settings from `.vscode/settings.json` in `.devcontainer/devcontainer.json`. i can investigate

Reply:
@maestromac do you know what version manager rubyLsp uses for `devcontainer` ?

Reply:
hey @maestromac wanted to give an update here. tl;dr - still working on a solution

- i tried setting `"rubyLsp.rubyVersionManager": "none"` in devcontainer.json , but this still fails with a `Failed to activate mise environment` error, as if the setting in devcontainer.json isn't being read / is being overwritten by the top-level settings.json 
- i noticed there's a prerelease of rubyLsp available, i switched to it (v0.6.10) thinking the devcontainer setting bug may have been resolved there. this release (see code in [Implement internal Ruby activation mechanism](https://github.com/Shopify/vscode-ruby-lsp/pull/923)) actually removes the `rubyVersionManager` setting in favor of `rubyExecutablePath`, which seems promising ! haven't been able to set the path value successfully yet. i added a comment to the PR in hopes of getting some guidance, see https://github.com/Shopify/vscode-ruby-lsp/pull/923#issuecomment-2035393649

Reply:
What about the default `auto` feature?

Reply:
No, setting `"rubyLsp.rubyVersionManager": "auto"` in devcontainer.json throws the same "Failed to activate mise" error as `"rubyLsp.rubyVersionManager": "none"`:
![Screen Shot 2024-04-05 at 5 16 02 PM](https://github.com/forem/forem/assets/13890477/e5abb68b-3c00-4cfb-99d0-024055e79815)

Reply:
OH you probably meant setting `"rubyLsp.rubyVersionManager": "auto"` in the top-level settings.json? In which case, yes! that works 🥳. will update the PR